packages/space-opera/src/components/animation_controls/animation_controls.ts

```diff
@@ -88,39 +84,32 @@ export class AnimationControls extends ConnectedLitElement {
           <me-checkbox id="animation-autoplay" label="Autoplay"
             ?checked="${!!this.config.autoplay}"
             @change=${this.onAutoplayChange}></me-checkbox>
-          <me-checkbox id="animation-play" label="Play animation"
-            ?checked="${!!this.playAnimation}"
-            @change=${this.onPlayAnimationChange}></me-checkbox>
         </div>
       </me-expandable-tab>
         `;
   }
 
   onAutoplayChange() {
-    dispatchAutoplayEnabled(this.autoplayCheckbox!.checked);
+    reduxStore.dispatch(
+        dispatchAutoplayEnabled(this.autoplayCheckbox!.checked));
   }
 
   onAnimationNameChange(event: CustomEvent) {
     // Autoplay must be enabled otherwise the animation won't play and the
     // console throws a warning.
     const dropdown = event.target as Dropdown;
     const value = dropdown.selectedItem?.getAttribute('value') || undefined;
-
-    dispatchAnimationName(value);
+    if (!(value && this.animationNames.indexOf(value) === -1)) {
```

Comment:
This condition was previously in the action, moved outside of it, as actions are supposed to contain as little logic as possible and should always return a value to be dispatched.

Reply:
yeah, oddly enough I think this was the logic before as well, I don't think this is a condition that can really fail though, because these values are limited to the animationNames that are inputted from the original model